### ISSUE TITLE
Incorrect default hyper-parameters in the code?

### ISSUE THREAD

**Issue by f90** [locuslab/TCN#32]
I am trying to reproduce the results for the polyphonic music (Nott dataset), but I am having trouble with setting the correct TCN hyperparams. Are the ones shown in Table 2 of the paper the ones I should use? If so, then why are the default parameters set differently, e.g. kernel size to 5 instead of 6 for Nott (or 3 for JSB)?

Also, when I manually set the parameters as shown in the paper, by using

```
--dropout 0.2 --clip 0.4 --ksize 6 --levels 4 --nhid 150
```
as parameters, then the TCN model has 2M parameters, but in the paper the model size is given as 1M (roughly). So now I am confused whether this is actually the setting used to produce the paper results?

**Comment by f90** [comment]
In general, it would be great to have a text file containing all the sets of commands needed to reproduce the paper results, e.g. python music_test.py --dropout x --clip y etc. for each experiment

**Comment by jerrybai1995** [comment]
In general, the default `argparse` hyperparameters in the repo should be up-to-date to produce the best outcome on the default dataset(s). But since polyphonic music datasets are relatively small, usually some basic tuning on kernel sizes and hidden sizes could be beneficial. I'll try to see if I can add a text file for reproducing the results (thanks for the advice :-)).

**Comment by f90** [comment]
So you mean the default arguments are optimised further than the ones in the paper, to provide even better results? While that's nice, I kind of assumed the defaults were identical to the paper ones. So yeah, providing the commands to reproduce the paper specifically would be cool! Thanks!
